src/overlays/ovl_i2/fox_me.c

```diff
@@ -1,97 +1,2527 @@
+                    if (boss->obj.pos.x > 0.0f) {
+                        boss->fwork[14] = -7.0f;
+                    }
+                    boss->timer_056 = (s32) (Rand_ZeroOne() * 40.0f) + 50.0f;
+                    boss->swork[17] = 1;
+                }
+                break;
+
+            case 1:
+                if (boss->fwork[14] < boss->vel.x) {
+                    boss->vel.x = boss->vel.x - 0.2f;
+                }
+                if (boss->vel.x < boss->fwork[14]) {
+                    boss->vel.x += 0.2f;
+                }
+                if (boss->timer_056 == 0) {
+                    boss->swork[17] = 0;
+                    boss->timer_056 = (s32) (Rand_ZeroOne() * 40.0f) + 50.0f;
+                }
+                break;
+        }
+    }
+
+    if (boss->swork[18] != 0) {
+        s32 id;
+
+        if (func_8008AC54(0) >= 4) {
+            id = 191;
+        } else {
+            id = 190;
+        }
+
+        Audio_PlaySfx(0x2903201BU, &boss->sfxPos, 4U, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+
+        src.x = 0.0f;
+        src.y = 330.0f;
+        src.z = -1022.0f;
+
+        Matrix_MultVec3fNoTranslate(gCalcMatrix, &src, &dest);
+
+        if (boss->swork[18] == 1) {
+            func_i2_80187D08(boss->obj.pos.x + dest.x, boss->obj.pos.y + dest.y, boss->obj.pos.z + dest.z, 50.0f,
+                             270.0f, 0.0f, 0x1E, 0, id);
+        } else {
+            func_i2_80187D08(boss->obj.pos.x + dest.x, boss->obj.pos.y + dest.y, boss->obj.pos.z + dest.z, 50.0f, 0.0f,
+                             0.0f, 0, 0, id);
+        }
+
+        src.y = -330.0f;
+
+        Matrix_MultVec3fNoTranslate(gCalcMatrix, &src, &dest);
+
+        if (boss->swork[18] == 1) {
+            func_i2_80187D08(boss->obj.pos.x + dest.x, boss->obj.pos.y + dest.y, boss->obj.pos.z + dest.z, 50.0f, 90.0f,
+                             0.0f, 0x1E, 0, id);
+        } else {
+            func_i2_80187D08(boss->obj.pos.x + dest.x, boss->obj.pos.y + dest.y, boss->obj.pos.z + dest.z, 50.0f, 0.0f,
+                             0.0f, 0, 0, id);
+        }
+        boss->swork[18] = 0;
+    }
+
+    if ((sp7C < 200.0f) && (sp7C > -1500.0f)) {
+        D_801784F8 = 0.0f;
+        D_801784FC = 150.0f;
+        D_80178500 = 0.0f;
+    }
+
+    switch (boss->actionState) {
+        case 0:
+            boss->swork[0] = 10;
+            boss->swork[1] = 10;
+
+            boss->obj.pos.y += ((0.0f - boss->obj.pos.y) * boss->fwork[1]);
+            boss->vel.z = boss->fwork[0] - D_80177D08;
+
+            if (sp7C < boss->fwork[9]) {
+                boss->actionState = 1;
+                boss->fwork[1] = 0.0f;
+            }
+
+            if (sp7C < -1000.0f) {
+                if (boss->fwork[1] < 0.005f) {
+                    boss->fwork[1] = boss->fwork[1] + 0.0001f;
+                }
+            }
+            break;
+
+        case 1:
+            boss->obj.pos.y += ((0.0f - boss->obj.pos.y) * boss->fwork[1]);
```

Comment:
```suggestion
            boss->obj.pos.y += - boss->obj.pos.y* boss->fwork[1])
```
I take it this doesn't match.

Reply:
Nope, it doesn't